pkg/plugins/plugins.go

```diff
@@ -140,7 +140,11 @@ func (pm *PluginManager) Init() error {
 		if p.IsCorePlugin {
 			p.Signature = PluginSignatureInternal
 		} else {
-			p.Signature = getPluginSignatureState(pm.log, p)
+			var err error
```

Comment:
technically I think you could remove this line, but just style so up to you.

Reply:
@kylebrandt I would have to try, I'm not sure if `err` is defined otherwise.

Reply:
@kylebrandt Oh yeah, it does work since `err` is defined already in the function.